packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -66,25 +73,20 @@ export default class Dropdown extends Component {
           theme.Dropdown,
           className
         )}
+        {...props}
       >
-        <Anchor
-          data-tid="dropdown-anchor"
-          {...props}
+        <Field
```

Comment:
Why are we changing the default anchor to a field? I feel like a button makes more sense as the default.

Reply:
Well.. the thought was for the search bar, but yeah I think you're right, let me change it back.